docs/guides/menus.md

```diff
@@ -95,6 +95,27 @@ $menu = Timber::get_menu( 'primary' );
 
 In the same way that you [can’t instantiate post objects directly](https://timber.github.io/docs/guides/posts/#extending-timber-post), you **can’t** instantiate `Timber\Menu` or `Timber\MenuItem` objects or an object that extends this class with a constructor. Timber will use the [Menu Class Map](https://timber.github.io/docs/guides/class-maps/#the-menu-class-map) and the [MenuItem Class Map](https://timber.github.io/docs/guides/class-maps/#the-menuitem-class-map) to sort out which class it should use.
 
+## Pages Menu
+
+Timber includes a function to create menu from your pages, without having to register menus first. This is a quick way to create a menu if you have a small website with only a couple of pages.
+
+```php
+$menu = Timber::get_pages_menu();
+```
+
+This function will return an instance of `Timber\PagesMenu`, which is not quite the same as `Timber\Menu`, but it contains the same `Timber\MenuItem` objects as you know them.
+
+If you want to extend a pages menu, you would do it like this:
+
+
+```php
+class CustomPagesMenu extends Timber\PagesMenu {
```

Comment:
Using the fully qualified `\Timber\PagesMenu` is safer - it'll still work inside a custom namespace. Maybe we should also name this custom class `ExtendedPagesMenu` to reinforce the hook example?

Reply:
Added in bb2fafe795b81826ff87d2b4bdc1512e567aeda3 👍 I should probably update to fully qualified namespaces in other places, too.